glvis.cpp

```diff
@@ -703,14 +703,16 @@ void ExecuteScriptCommand()
          int pal;
          scr >> pal;
          cout << "Script: palette: " << pal << endl;
-         paletteSet(pal-1);
+         vs->GetPalette().SetPalette(pal);
```

Comment:
`pal` or `pal-1` here?

Reply:
I think this should be the original `pal-1` value. Fixed.